web/test/step_definitions/browser.js

```diff
@@ -1,9 +1,13 @@
-import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
+import {
+  Given,
+  Then
+} from 'cypress-cucumber-preprocessor/steps';
 
 const url = Cypress.config('baseUrl');
+let row;
```

Comment:
Cypress recommends not to do this, but to use their `alias` functionality.. 

```typescript
.as('row')
```

https://docs.cypress.io/guides/core-concepts/variables-and-aliases.html#Sharing-Context

Reply:
ok